Index: src/main/java/com/yd/util/DatabaseConnection.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.yd.util;\n\nimport java.sql.Connection;\nimport java.sql.DriverManager;\nimport java.sql.SQLException;\n\npublic class DatabaseConnection {\n\n    private static final String URL = \"jdbc:mysql://localhost:3306/INSTARGRAM?useSSL=false&serverTimezone=UTC\";\n    private static final String USER = \"root\";\n    private static final String PASSWORD = \"yd157238+\";\n\n    private static Connection connection = null;\n\n    public static Connection getConnection() throws SQLException {\n        if (connection == null || connection.isClosed()) {\n            try {\n                Class.forName(\"com.mysql.cj.jdbc.Driver\");\n                connection = DriverManager.getConnection(URL, USER, PASSWORD);\n            } catch (ClassNotFoundException | SQLException e) {\n                e.printStackTrace();\n            }\n        }\n        return connection;\n    }\n\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/yd/util/DatabaseConnection.java b/src/main/java/com/yd/util/DatabaseConnection.java
--- a/src/main/java/com/yd/util/DatabaseConnection.java	(revision 01d84f02871a2efa00040b959259cceb24ce61b4)
+++ b/src/main/java/com/yd/util/DatabaseConnection.java	(date 1731908467690)
@@ -6,7 +6,7 @@
 
 public class DatabaseConnection {
 
-    private static final String URL = "jdbc:mysql://localhost:3306/INSTARGRAM?useSSL=false&serverTimezone=UTC";
+    private static final String URL = "jdbc:mysql://localhost:3306/INSTARGRAM?allowPublicKeyRetrieval=true&useSSL=false&serverTimezone=UTC";
     private static final String USER = "root";
     private static final String PASSWORD = "yd157238+";
 
